programs/svm-spoke/src/utils/transfer_utils.rs

```diff
@@ -0,0 +1,29 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token_interface::{transfer_checked, Mint, TokenAccount, TokenInterface, TransferChecked};
+
+use crate::State;
+
+pub fn transfer_from<'info>(
```

Comment:
this is great. it's similar to how we were doing transfers previously, but now wrapped in the `TransferChecked` block.